pubspec.yaml

```diff
@@ -37,6 +37,8 @@ dependencies:
   firebase_core: ^2.17.0
   firebase_crashlytics: ^3.3.7
   firebase_analytics: ^10.5.1
+  firebase_remote_config: ^4.3.2
+  package_info: ^2.0.2
```

Comment:
why package_info ?

Reply:
To get version of the app

Reply:
we have package_info_plus already i believe.